cc-ui/src/pages/Login.tsx

```diff
@@ -109,21 +112,26 @@ export default function Login() {
 
                     <FormLabel 
                         id="wrongCred" 
-                        hidden={true}
-                        color='#EE0004' 
+                        color='#EE0004'
+                        hidden={!wrongCredState}
                         >
-                            Virheellinen käyttäjätunnus tai salasana.</FormLabel>
+                            Virheellinen käyttäjätunnus tai salasana.
+                    </FormLabel>
 
                     <Button 
                         type="submit"
                         textTransform='uppercase'
                         colorScheme='blue'
+                        onClick={navigateHome}

```

Comment:
Button type is submit, so onClick should trigger the onSubmit -function defined in the form element, so use that and remove this